src/components/organizations/show/facts.js

```diff
@@ -0,0 +1,161 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+
+import Icon from 'react-fa'
+
+import Histogram from 'gComponents/simulations/histogram/index'
+
+import {simulateFact} from 'gEngine/facts'
+import {addStats} from 'gEngine/simulation'
+
+import './facts.css'
+
+
+// TODO(matthew): Also add validations for values on the server.
+
+const FactRow = ({fact}) => (
+  <div className='Fact'>
+    <div className='row'>
+      <div className='col-md-3'>
+        <Histogram
+          height={30}
+          simulation={fact.simulation}
+          cutOffRatio={0.995}
+        />
+      </div>
+      <div className='col-md-6'><span className='name'>{fact.name}</span></div>
+      <div className='col-md-2'>
+        <div className='variableName'>
+          <span className='prefix'>#</span>
+          <span className='variable'>{fact.variable_name}</span>
+        </div>
+      </div>
+      <div className='col-md-1'>
+        <span className='ui button options'><Icon name='ellipsis-v' /></span>
+      </div>
+    </div>
+  </div>
+)
+
+const hasAllNonEmpty = (obj, props) => !_.some(props.map(prop => _.isEmpty(_.get(obj, prop))))
```

Comment:
why not ``_.all`` instead? (but the inverse)

this could also be split into two lines or something, quite terse.